app/labor/assign_tag_moderator.rb

```diff
@@ -48,4 +48,8 @@ def self.remove_tag_moderator(user, tag)
     Rails.cache.delete("user-#{user.id}/tag_moderators_list")
     MailchimpBot.new(user).manage_tag_moderator_list
   end
+
+  def self.chat_channel_slug(tag)
+    tag.mod_chat_channel&.slug
```

Comment:
We need to pass the chat channel slug through the email now if there is one to point the user to the new channel that they were added to